### DISCUSSION TITLE
Phase banner

### DISCUSSION THREAD

**Discussion by jrbarnes9** [ONSdigital/design-system#8] [Feedback]
Use this place to discuss the [phase banner component in the ONS Design System](https://ons-design-system.netlify.com/components/phase-banner/)

**Comment by armstrongb** [comment]
Requires addition to the documentation that different phases don't have different colours; Alpha and Beta phases are all black. I think it should be stated to document that this was a conscious decision and should not permit additional colours at this time without strong evidence of a user need.

**Reply by jrbarnes9** [reply]
Addressed in #1600

**Comment by jrbarnes9** [comment]
Phase banner badge text shouldn't wrap and banner content should be vertically aligned to top

<img width="330" alt="Screenshot 2021-07-23 at 09 12 50" src="https://user-images.githubusercontent.com/43346934/126754781-01d6a674-6b75-46ba-84d2-f735201031f4.png">

**Reply by jrbarnes9** [reply]
Fixed in #1600

**Comment by jrbarnes9** [comment]
Phase banner recommended content link text should be more descriptive out of context and at the end of the sentence. e.g. "This is a new service. To help us improve it, please [give us your feedback](/#0)"

**Reply by jrbarnes9** [reply]
#1601 raised

**Reply by alessioventuriniAND** [reply]
fixed by #1985

**Comment by jrbarnes9** [comment]
We should allow services using the phase banner to make the feedback link open in a new tab (with the required attributes) to prevent users information being lost when giving feedback during a transaction on a live service.

**Reply by jrbarnes9** [reply]
Raised #1984

**Reply by alessioventuriniAND** [reply]
Fixed by #1985

**Comment by jrbarnes9** [comment]
Phase banner shares same background as cookie banner

There is no distinct division
![Screenshot 2022-07-14 at 15 09 29](https://user-images.githubusercontent.com/43346934/179002631-9d079fda-4be5-4777-803e-4fc83b894cfe.png)

**Reply by jrbarnes9** [reply]
Raised #2325

**Comment by LaurenDowling** [comment]
Looking at the HTML for this component, the BETA itself is wrapped in a H3 tag. Given its position on the page, this means (from what I could make out on Voiceover) that it appears as the first heading. Would this potentially be confusing for users flicking through the headers to just hear 'Beta' and nothing else? 
Within the GOV.UK Design System, they seem to use a strong tag instead. Consequently it's not read out when navigating through the various headers on the page.

**Reply by jrbarnes9** [reply]
Raised #2462
